RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -554,14 +554,13 @@ def __loadConfigFileLevel2(self):
         
         try:
             defualtCfg = oSuiteConfig['default']['name']
+            self.sTestCfgFile = oSuiteConfig[self.sConfigName]['name']
+            sTestCfgDir = oSuiteConfig[self.sConfigName]['path']
         except:
-            logger.error("Testsuite management is in configuration level2.")
-            logger.error("The file '%s' has no default config or wrong testsuite config format for level 2" %(self.sTestSuiteCfg))
-            logger.error("Testsuite will run with default config file: %s " %(self.sTestCfgFile))
+            CConfig.sLoadedCfgError = "Testsuite management - Loading configuration level 2 is failed! \n \
+                The vairant '%s' is not defined in '%s'" % (self.sConfigName, os.path.abspath(self.sTestSuiteCfg))
```

Comment:
Hi Son,
in general the rule is to use f-strings! variant has a typo.
```
The **variant** {self.sConfigName} ....
```
Thank you,
 Thomas

Reply:
Updated